precompiles/src/secp256k1.rs

```diff
@@ -135,6 +135,28 @@ pub mod tests {
         solana_secp256k1_program::{new_secp256k1_instruction, DATA_START},
     };
 
+    fn test_verify_with_alignment(
```

Comment:
can we move this up to a common module under dcou?

Reply:
yeah sure. will have to pass in verify fn - seems fine

Reply:
5eeff4e0fa lmk what you think. Just put it under `#[cfg(test)]` since it doesn't really need dcou.

Reply:
ah right 'cause we'r not crossing crate boundary. wfm!